tests/ImportCommentsPreserved/__snapshots__/ppsi.spec.ts.snap

```diff
@@ -0,0 +1,178 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`import-comments-babel-pulling-single-line-bug.ts - typescript-verify > import-comments-babel-pulling-single-line-bug.ts 1`] = `
+
+import B from "b";
+// Trailing comment (not supposed to be on same line)
+// Bottom of imports single-line comment
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+import B from "b";
+// Trailing comment (not supposed to be on same line)
+
+// Bottom of imports single-line comment
+
+`;
+
+exports[`import-comments-document-top-test1.ts - typescript-verify > import-comments-document-top-test1.ts 1`] = `
+// This is part of multiple-lines at document-top
+// Second line followed by a gap
+
+import b from "b";
+import a from "a";
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// This is part of multiple-lines at document-top
+// Second line followed by a gap
+
+import a from "a";
+import b from "b";
+
+`;
+
+exports[`import-comments-document-top-test2.ts - typescript-verify > import-comments-document-top-test2.ts 1`] = `
+/**
+ * This is part of multiple-lines at document-top
+ * Second line of text, comment is followed by a gap (gap gets eaten)
+ */
+
+import b from "b";
+import a from "a";
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/**
+ * This is part of multiple-lines at document-top
+ * Second line of text, comment is followed by a gap (gap gets eaten)
+ */
+
+import a from "a";
+import b from "b";
+
```

Comment:
Any idea why there's a newline here?

Reply:
I think when imports get shuffled, one newline escapes and gets inserted below the imports (excess newlines are trimmed).

I'm not sure on this one. Is it a blocking issue?